pkgs/applications/networking/p2p/transmission/4.nix

```diff
@@ -37,14 +37,17 @@
 , apparmorRulesFromClosure
 }:
 
-stdenv.mkDerivation rec {
+let
+  # Needs macOS >= 10.14.6
+  stdenv' = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
```

Comment:
why not just use `darwin.apple_sdk_11_0.callPackage` in `all-packages.nix`?

iirc it's a noop on linux etc, anyways and that way, it can be more easily removed, once it becomes the default.

Reply:
> why not just use `darwin.apple_sdk_11_0.callPackage` in `all-packages.nix`?
> 
> iirc it's a noop on linux etc, anyways and that way, it can be more easily removed, once it becomes the default.

Sounds good, let's try that.